index.html

```diff
@@ -2205,47 +2189,56 @@ <h2>Sessions</h2>
  context</a>, when changing browsing contexts.</p>
 
-<p>
-A <a>session</a> has an associated <dfn data-lt="session script timeout|session page load timeout|session implicit wait timeout">session timeouts</dfn>
-that records the timeout duration values used to control the behavior
+<p>An <a>HTTP session</a> has an associated <dfn data-lt="session
+script timeout|session page load timeout|session implicit wait
+timeout">session timeouts</dfn> that records the timeout duration
+values used to control the behavior
 of <a href=#executing-script>script evaluation</a>,
 <a href=#navigation>navigation</a>,
 and <a href=#elements>element retrieval</a>.
 
-<p>A <a>session</a> has an associated <dfn>page loading strategy</dfn>,
- which is one of the keywords from the <a>table of page load strategies</a>.
- Unless stated otherwise, it is <a>normal</a>.
-
-<p>A <a>session</a> has an associated <dfn>secure TLS</dfn> state
- that indicates whether untrusted or self-signed TLS certificates
- should be trusted for the duration of the WebDriver session.
- If it is unset, this indicates that certificate- or TLS errors
- that occur upon <a>navigation</a> should be suppressed.
- The state can be unset by providing
- an "<code>acceptInsecureCerts</code>" <a>capability</a> with the value true.
- Unless stated otherwise, it is set.
+<p>An <a>HTTP session</a> has an associated <dfn>page loading
+ strategy</dfn>, which is one of the keywords from the <a>table of
+ page load strategies</a>.  Unless stated otherwise, it
+ is <a>normal</a>.
 
-<p>A <a>session</a> has an associated <a>strict file interactability</a>
- state which is a boolean.
+<p>An <a>HTTP session</a> has an associated <a>strict file
+ interactability</a> state which is a boolean.
 
-<p>
-A <a>session</a> has an associated <a>user prompt handler</a>.
+<p>A <a>HTTP session</a> has an associated <a>user prompt handler</a>.
 Unless stated otherwise it is in the <a>dismiss and notify state</a>.
 
-<p>A <a>session</a> has an associated list of <a>active input sources</a>.
+<p>An <a>HTTP session</a> has an associated <dfn>request queue</dfn> which is a
+ <a>queue</a> of <a data-lt=request>requests</a> that are currently awaiting
+ processing.
 
-<p>A <a>session</a> has an associated <a>input state table</a>.
+<h3>Global State</h3>
 
-<p>A <a>session</a> has an associated <a>input cancel list</a>.
+<p>In addition to per-session state, a <a>remote end</a> that is
+an <a>endpoint node</a> also has additional state that is global
+across all sessions.</p>
 
-<p>A <a>session</a> has an associated <dfn>request queue</dfn> which is a
- [=queue=] of [=requests=] that are currently awaiting
- processing.
+<p>An <a>endpoint node</a> has an associated <dfn>accept insecure
+ TLS</dfn> flag that indicates whether untrusted or self-signed TLS
+ certificates are treated as trusted. The default value of the flag is
+ false if the endpoint doesn't support accepting insecure TLS
+ connections, or unset otherwise.</p>
```

Comment:
So when do we actually allow to set this global state? I assume the first created session will have to install the handler, and when the last session gets closed it will be removed?

Same question applies for other global states.